tests/robot-tests/tests/admin/analyst/edit_and_approve_release_as_publication_release_approver.robot

```diff
@@ -0,0 +1,139 @@
+*** Settings ***
```

Comment:
I think this file should live in `tests/admin_and_public` since it does visit the public frontend at the end.

Or maybe you could remove the bit that checks the public frontend - not sure how much value that provides?

Reply:
Yeah I agree, good shout.  I think I'll just remove it and just make sure we're checking that the Release status makes it to Published (which we already do)